packages/fxa-settings/src/components/LinkDamaged/en.ftl

```diff
@@ -0,0 +1,12 @@
+## LinkDamaged component
+
+# The user followed a link to reset their password that was received by email
+# but the link was damaged (for example mistyped or broken by the email client).
+password-reset-link-damaged-header = Reset password link damaged
+
+# The user followed a link to signin that was received by email
+# but the link was damaged (for example mistyped or broken by the email client).
+signin-link-damaged-header = Confirmation link damaged
+
+# The user followed a link received by email, but the link was damaged.
+link-damaged-message = The link you clicked was missing characters, and may have been broken by your email client. Copy the address carefully, and try again.
```

Comment:
Is there a reason this needs a new ID? Since these have already been localized and string isn't changing the ID could be left as-is and existing translations could be used.